### ISSUE TITLE
Host event IDs are not globally unique

### ISSUE THREAD

**Issue by paulotfilho** [naparuba/shinken#1836]
Tested this with Shinken 2.4.2 on Debian 8.

[Shinken docs](http://shinken.readthedocs.org/en/latest/05_thebasics/macrolist.html#hosteventid) for standard macros say that $HOSTEVENTID$ and $HOSTPROBLEMID$ are globally unique numbers associated with the host’s current state/problem state, incremented by one every time a host (or service) experiences a state change/transitions from an UP or OK state to a problem state. 

The problem is that this is not how Shinken actually works, because I have experienced multiple cases where a host had the same event ID as another host in an older event. It's like the event ID is actually local to the host, not globally unique to all hosts.

**Comment by renankof** [comment]
+1

**Comment by naparuba** [comment]
True. they are standard int, we should change them to uuid, will be naturally unique.
